### DISCUSSION TITLE
Uploading C# files

### DISCUSSION THREAD

**Discussion by Nevrai** [danny-avila/LibreChat#4257] [Help Wanted]
Hi. I submitted issue #4056 two weeks ago. My primary use case for LibreChat is C# programming, so it’s a shame it doesn’t support C# file uploading. However, @danny-avila said that LibreChat is meant to support C# files.

I was wondering: is a fix being worked on, and do I have to wait for an update to LibreChat? Or is there anything I can change in the files in order to make uploading C# files work?

**Comment by danny-avila** [comment]
You just need to update, and it works.

**Comment by Nevrai** [comment]
Thanks. It works now. I don’t think I updated, though. I still have the same LibreChat version, and there are no recent updates in the [changelog](https://www.librechat.ai/changelog). But I’m happy it works! I can now finally upload C# files!
